lib/insert-asset-manifest.js

```diff
@@ -0,0 +1,22 @@
+var mergeTrees = require('broccoli-merge-trees');
+var AssetManifestInserter = require('./asset-manifest-inserter');
+
+/**
+ * Given a tree, this function will insert the asset manifest into index.html.
+ *
+ * The `options` object is passed directly into the broccoli plugin.
+ *
+ * @public
+ * @param {Tree} tree
+ * @param {object} options
+ * @return {Tree}
+ */
+module.exports = function insertAssetManifest(tree, options) {
+  options = options || {};
+
+  options.cacheInclude = options.cacheInclude || ['index.html', 'tests/index.html', 'asset-manifest.json'];
```

Comment:
We shouldn't necessarily have it configurable but doing it "right" is also ridiculous because of these:

- https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js#L637-L659
- https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js#L751-L765

Going to decide how much of this actually makes sense to support.

(Note that I thought of this only after your comment. Original reason was because that's how my brain automatically handles options.)